DataFormats/TrackerCommon/interface/LegacyTrackerTopology.h

```diff
@@ -0,0 +1,93 @@
+#ifndef TRACKER_TOPOLOGY_LEGACY_H
+#define TRACKER_TOPOLOGY_LEGACY_H
+
+#include "DataFormats/TrackerCommon/interface/TrackerTopology.h"
+#include "DataFormats/DetId/interface/DetId.h"
+
+namespace LegacyTrackerTopology {
+  
+  inline static std::unique_ptr<TrackerTopology> getTrackerTopology()
+  {
+        TrackerTopology::PixelBarrelValues pxb;
+        pxb.layerStartBit_ =  16;
+        pxb.ladderStartBit_ = 8;
+        pxb.moduleStartBit_ = 2;
+        pxb.layerMask_ =  0xF;
+        pxb.ladderMask_ = 0xFF;
+        pxb.moduleMask_ = 0x3F;
+
+        TrackerTopology::PixelEndcapValues pxf;
+        pxf.sideStartBit_ =   23;
+        pxf.diskStartBit_ =   16;
+        pxf.bladeStartBit_ =  10;
+        pxf.panelStartBit_ =  8;
+        pxf.moduleStartBit_ = 2;
+        pxf.sideMask_ =   0x3;
+        pxf.diskMask_ =   0xF;
+        pxf.bladeMask_ =  0x3F;
+        pxf.panelMask_ =  0x3;
+        pxf.moduleMask_ = 0x3F;
+
+        TrackerTopology::TECValues tecVals;
+        tecVals.sideStartBit_ =        18;
+        tecVals.wheelStartBit_ =       14;
+        tecVals.petal_fw_bwStartBit_ = 12;
+        tecVals.petalStartBit_ =       8;
+	tecVals.ringStartBit_ =        5;
```

Comment:
there seems to be something weird with the indentation here... maybe a mix of tabs and spaces?